doc/modules/model_evaluation.rst

```diff
@@ -92,7 +92,7 @@ Scoring                                Function
 
 **Regression**
 'explained_variance'                   :func:`metrics.explained_variance_score`
```

Comment:
Can you keep the same horizontal alignment as the line before and the line before
```suggestion
'neg_max_error'                        :func:`metrics.max_error`
```

Reply:
Not sure how that happened! Should be fixed now. Thanks for catching it.